tests/next_tests/integration_tests/feature_tests/cases.py

```diff
@@ -173,9 +174,11 @@ def field(
         svals = tuple(sizes.values())
         n_data = int(np.prod(svals))
         self.start += n_data
-        return gtx.np_as_located_field(*sizes.keys())(
-            np.arange(start, start + n_data, dtype=dtype).reshape(svals)
-        )
+        if dtype == "bool":
+            f = np.random.choice(a=[False, True], size=(svals))
+        else:
```

Comment:
I am not sure if UniqueInitializer should work for bools, because there is no way to make each item unique. This should probably be in a separate initializer class (I think it's ok to have to pass it manually when initializing a bool field in tests). UniqueInitializer should probably just throw an error saying that it can't do bools.